eng/scripts/TypeSpec-Generate-Sdk.ps1

```diff
@@ -0,0 +1,35 @@
+[CmdletBinding()]
+param (
+  [Parameter(Position = 0)]
+  [ValidateNotNullOrEmpty()]
+  [string] $SdkRepoRootDirectory,
+  [Parameter(Position = 1)]
+  [string] $TypeSpecProjectDirectory, # A directory of `tspconfig.yaml` or a remoteUrl of `tspconfig.yaml`
```

Comment:
Can we default this to "." so folks don't need to pass it when they inside the directory.

Reply:
Sure, I can set `.` as default path.